packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -222,13 +222,25 @@ const ValidDashboardChartTileMinimal: FC<{
     );
 };
 
-const InvalidDashboardChartTile: FC = () => (
-    <NonIdealState
-        title="No chart available"
-        description="Chart might have been deleted or you don't have permissions to see it."
-        icon="search"
-    />
-);
+const InvalidDashboardChartTile: FC<
+    Pick<DashboardChartTileMainProps, 'tile'>
+> = ({ tile }) => {
+    const newChartName = sessionStorage.getItem('newChartName');
+    console.log(newChartName);
+    return tile.properties.belongsToDashboard ? (
```

Comment:
I would change this to `tile.properties.newChartData` and use `tile.properties.newChartData.name` instead of passing it via storage. 
Using storage won't work if they add 2 charts.

Reply:
I tried but I had a type error and this was my fix 😭

Reply:
https://github.com/lightdash/lightdash/issues/6450